frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -112,7 +145,7 @@ export const valuesToApiFormat = (
 const getOwnedParcels = (leasedLand: ILeasedLand[], parcels: IParcel[]): IParcel[] => {
   const ownedParcels: IParcel[] = [];
   leasedLand.forEach((ll: ILeasedLand, index: number) => {
-    if (ll.type === LeasedLand.owned) {
+    if (ll?.type === LeasedLand.owned && parcels[index]) {
```

Comment:
what does adding the ? do here? I see it a lot

Reply:
This is the null propagation operator. Basically this allows an expression to short circuit safely if a value is null or undefined.
```
Example: 
const test = { field: undefined }
const newValue = test.field.someValue // this would throw a null reference exception
const safeNewValue = test.field?.someValue // if field is null or undefined, return null or undefined instead of continuing to evaluate the expression
```